applications/nrf_desktop/README.rst

```diff
@@ -1857,6 +1857,46 @@ These are valid for events that have many listeners or sources, and are gathered
    doc/hfclk_lock.rst
    doc/event_rel_modules.rst
 
+API documentation
```

Comment:
The "API documentation" section could be after the "Application configuration options" section.

Reply:
moved